src/analysis/modals/CloudEnvironmentModal.ts

```diff
@@ -460,14 +460,16 @@ export const CloudEnvironmentModal = ({
         () => {
           return {
             ...baseProps,
-            href:
-              app &&
-              (cloudProvider === cloudProviderTypes.AZURE
-                ? app?.proxyUrls['cbas-ui']
-                : app?.proxyUrls['cromwell-service']),
+            href: app && (cloudProvider === cloudProviderTypes.GCP ? app?.proxyUrls['cromwell-service'] : ''),
```

Comment:
I'm leaving the `href` value here for GCP workspaces because I don't want to cause unintended side effects with this (probably) short-lived change.

Reply:
Would it be better to leave this consistent instead of having some routing via `href` and some routing via `onClick`? You could use `Nav.getLink` to supply the href property.

Reply:
I'm not sure - one thing that makes me pause is that the behaviors are slightly different. Maybe it's easy enough to replicate the `href` behavior in the `onClick` callback, but I was trying to change as little as possible.

Reply:
(Specifically the difference in behavior is that `href` routing opens a new tab by default, which I'm trying to avoid in Azure workspaces)